core/class/jMQTT.class.php

```diff
@@ -2110,33 +2126,37 @@ public function publish($cmdName, $topic, $payload, $qos, $retain) {
 			//   * If payload contains ", they are backslashed \"
 			// Fix #110
 			// Since Core commit https://github.com/jeedom/core/commit/430f0049dc74e914c4166b109fb48b4375f11ead
-			// payload can be more than int/bool/string
+			// payload can become more than int/bool/string
 			$payload = json_encode($payload, JSON_UNESCAPED_UNICODE);
 		}
-		$payloadLogMsg = ($payload === '') ? '(null)' : $payload;
-		if (log::getLogLevel(__CLASS__) > 100)
-			$this->log('info', sprintf(__("Cmd #%1\$s# -> %2\$s sur le topic '%3\$s'", __FILE__), $cmdName, $payloadLogMsg, $topic));
-		else
-			$this->log('info', sprintf(__("Cmd #%1\$s# -> '%2\$s' sur le topic '%3\$s' (qos=%4\$s, retain=%5\$s)", __FILE__), $cmdName, $payload, $topic, $qos, $retain));
-		$broker = $this->getBroker();
+		$payloadLogMsg = ($payload === '') ? '\'\' (null)' : "'".$payload."'";
```

Comment:
the idea was to put single quote around, no? :-)
'\'\' (null)' => '\'(null)\''

Reply:
Actually no, to avoid confusing an empty payload with text "(null)" payload